google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/SourceLocation.java

```diff
@@ -33,6 +33,14 @@
   private final Long line;
   private final String function;
 
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  public static Builder newBuilder(SourceLocation sourceLocation) {
```

Comment:
you should provide a non-static method toBuilder() instead that someone can call on `SourceLocation`.